infoTree/getURLfile.js

```diff
@@ -0,0 +1,18 @@
+function getURLindex(label){
+    //first find which URL maps to it
+    let targetURL = "";
+    for (int i = 0; i < whereDoesItGo.length; ++i){
```

Comment:
`whereDoesItGo` is no defined. This will be a runtime error

Reply:
This is a global in whereDoesItGo.js
I assume I have to require this variable then?

Reply:
Fixed using import in most recent update